### ISSUE TITLE
Issue when using Rollup with ObjectModel 3.4.3

### ISSUE THREAD

**Issue by minhtule** [sylvainpolletvillard/ObjectModel#64]
Hi, I have an issue when using Rollup to bundle my project which depends on ObjectModel. It doesn't work with the latest version 3.4.3 but it works with 3.4.2. I've created a demo project [here](https://github.com/minhtule/rollup-objectmodel-3.4.3).

In `main.js`, I import the `ObjectModel` like the following
```js
import { ObjectModel } from 'objectmodel';

const Foo = ObjectModel({
  foo: String,
});
```

Then I bundle `main.js` to an `iife` output using Rollup. 

There are circular dependency warnings when using Rollup to bundle ObjectModel. And version `3.4.3` has 1 extra warning compared to `3.4.2`, probably because of this [commit](https://github.com/sylvainpolletvillard/ObjectModel/commit/2db66d2b42da7c28275d0619d051e7a4769f9e71).

How should I resolve this issue?

**Comment by sylvainpolletvillard** [comment]
Ha, sorry about that, looks like I pushed in a hurry last night. Will try to fix this today, in the mean time please keep up with 3.4.2

**Comment by sylvainpolletvillard** [comment]
All circular dependencies issues should be resolved in v3.4.4, can you confirm the new version solves your problem ?

**Comment by minhtule** [comment]
@sylvainpolletvillard Yes, all the circular dependencies warnings are gone now and my build works with v3.4.4. Thank you very much for your quick response and update!
